server/datastore/cached_mysql/cached_mysql.go

```diff
@@ -0,0 +1,99 @@
+package cached_mysql
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	redigo "github.com/gomodule/redigo/redis"
+	"github.com/pkg/errors"
+)
+
+type cachedMysql struct {
+	fleet.Datastore
+
+	redisPool fleet.RedisPool
+}
+
+const (
+	CacheKeyAppConfig = "AppConfig"
```

Comment:
I think it'd be best to namespace the key a bit, just for clarity when inspecting redis keys and future use - e.g. `cache:AppConfig` or something like that (dbcache maybe?).